src/openms/include/OpenMS/CHEMISTRY/EnzymaticDigestion.h

```diff
@@ -179,6 +179,21 @@ namespace OpenMS
      */
     std::vector<int> tokenize_(const String& sequence, int start = 0, int end = -1) const;
 
+    /**
+      @brief Generates semi-specific digestion products
```

Comment:
will it skip fully specific variants?

will it take MC's into account?

Please document